docs/astro.config.mjs

```diff
@@ -10,6 +10,11 @@ const { openAPISidebarGroups, starlightOpenAPI } = await generateAPI([
     label: 'Open Payments',
     schema: '../openapi/resource-server.yaml'
   },
+  {
+    base: 'apis/wallet-address-server',
+    label: 'Wallet Addresses',
+    schema: '../openapi/wallet-address-server.yaml'
+  },
```

Comment:
We should update the link on lines 233 and 238 in this file to reference `/wallet-address-server` otherwise, we get a 404 on the "API specifications > Open Payments endpoints > wallet address" pages